internal/presenters/components.go

```diff
@@ -120,11 +129,18 @@ func RenderSummary(summary *json_schemas.TestSummary, orgName string, testPath s
 	openIssueLabelledCount := ""
 	ignoredIssueLabelledCount := ""
 
-	slices.Reverse(summary.SeverityOrderAsc)
+	minLevelPointer := slices.Index(summary.SeverityOrderAsc, severityMinLevel)
+	reversedSlice := reverseSlice(summary.SeverityOrderAsc)
 
-	for _, severity := range summary.SeverityOrderAsc {
+	for _, severity := range reversedSlice {
 		for _, result := range summary.Results {
+			satisfyMinLevel := slices.Index(summary.SeverityOrderAsc, result.Severity) >= minLevelPointer
 			if result.Severity == severity {
+				if !satisfyMinLevel {
+					openIssueLabelledCount += renderInSeverityColor(severity, fmt.Sprintf(" %d %s ", 0, strings.ToUpper(severity)))
+					ignoredIssueLabelledCount += renderInSeverityColor(severity, fmt.Sprintf(" %d %s ", 0, strings.ToUpper(severity)))
+					continue
```

Comment:
**question:** Design decision will be needed on whether we want to show `0 LOW` when in fact there were Low Issues but they have been filtered out by `--severity-threshold` setting.